app/hotels/src/gallery/GalleryGrid.js

```diff
@@ -106,6 +106,7 @@ export default class GalleryGrid extends React.Component<Props, State> {
           backdropColor="black"
           backdropOpacity={1}
           style={{ margin: 0 }}
+          onRequestClose={this.closeStripe}
```

Comment:
@tbergq I see that this is quite often a problem. What about fixing it more systematically and introduce required `onRequestClose` property in the `Modal` component? (therefore you really must implement it)